content/rs/developing/crdbs/_index.md

```diff
@@ -191,6 +191,13 @@ the syncer initiates full reconciliation with other peers in the CRDB.
 
 ## INFO
 
+The INFO command shows these statistics differently for CRDBs:
+
+- DBSIZE (in `shard-cli info`) reports key header instances, not unique keys.
+- The keys count in DBSIZE (in `shard-cli info`) can be less than Expires (in `bdb-cli info`)
+  because expires are not always removed when a key becomes a tombstone.
```

Comment:
maybe explain what is a tombstone? (a key that is logically deleted but still takes memory until it can be collected by the garbage collector)